portal/templates/login/we-have-sent-you-another-link.njk

```diff
@@ -11,7 +11,7 @@
 
   <div class="govuk-grid-row govuk-body">
     <div class="govuk-grid-column-three-quarters">
-      <p class="govuk-!-margin-bottom-6">We've sent you another email with a link to {{emailAddress}}.</p>
+      <p class="govuk-!-margin-bottom-6" data-cy="sign-in-link-target-email-address">We've sent you another email with a link to {{ signInLinkTargetEmailAddress }}.</p>
```

Comment:
Do we want to rename this var to something like `obscuredSignInLinkTargetEmailAddress`